ProjFS.Mac/PrjFSKext/PrjFSKext/PerformanceTracing.cpp

```diff
@@ -3,21 +3,18 @@
 #include <stdatomic.h>
 #include <IOKit/IOUserClient.h>
 
-PerfTracingProbe profile_probes[Probe_Count];
+PerfTracingProbe profile_probes[PrjFSPerfCounter_Count];
```

Comment:
With this line of code removed from PerformanceTracing.hpp:

```
extern PerfTracingProbe profile_probes[Probe_Count];
```
I think `profile_probes` is now local to this file and can be marked as `static`